lib/modules/platform/gitlab/index.spec.ts

```diff
@@ -3110,7 +3166,6 @@ describe('modules/platform/gitlab/index', () => {
           number: 1,
           prTitle: 'title',
           prBody: 'body',
-          state: 'closed',
```

Comment:
This shouldn't be missing

Reply:
Where is the switch from `closed` and `open` coming from?

Reply:
It should not be closed the test description says add/remove labels. No, logic to add/remove labels & close a pr so I changed it to `open`.